CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 7.1.0
```

Comment:
Does this need to be on a `7.x` branch? Master is currently on `8.1.1`. As of v8, `"@hapi/hapi": ">=18.3.1"` is required.

Reply:
Oops!  I'd originally based this commit on the last `7.x` release, but then later rebased it against master and forgot to fix this version.  I'll fix that.

I ultimately want this change available in both the `7.x` and `8.x` releases in order to support applications using Hapi 17 or 18.